### DISCUSSION TITLE
【与技术无关】能否发一下这张壁纸？

### DISCUSSION THREAD

**Discussion by superleeyom** [tw93/Pake#309] [Q&A]
特别喜欢这张壁纸，可不可以方便贴下原图呢，太感谢啦
![image](https://user-images.githubusercontent.com/22115219/218613178-c4293cfc-8462-4f57-bf76-97e67d82c210.png)

**Comment by tw93** [comment]
https://tw93.fun/images/background.png

**Reply by superleeyom** [reply]
谢谢❤️
